treeherder/services/pulse/consumers.py

```diff
@@ -206,5 +238,14 @@ def thd(consumer):
             t.join()
 
 
-def prepare_consumers(consumer_cls, sources, build_routing_key=None):
-    return Consumers([consumer_cls(source, build_routing_key) for source in sources])
+def prepare_consumers(listening_params):
```

Comment:
Once the ```single class``` approach is verified, this will be modified to serve better data!

Reply:
This is all confusing to me. Would you mind adding a test to verify that it does what is expected?

Reply:
Okay. Will do so!